pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -90,8 +96,8 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 		}
 	}
 
+	allInfos := infos
```

Comment:
is this still neccessary?

Reply:
the next line will remove all pod level cgroup from infos, so I save it here into allInfos